docusaurus.config.js

```diff
@@ -105,23 +105,27 @@ const config = {
             position: "left",
           },
           {
-            type: "doc",
-            label: "Build",
-            docId: "build/building-modules/intro",
-            position: "left",
+            type: 'dropdown',
+            label: 'Build',
+            position: 'left',
+            items: [
+              {
+                type: "doc",
+                label: "Build a Module",
```

Comment:
Once we have this: https://github.com/cosmos/cosmos-sdk-docs/issues/57 this will be better, but for now, I guess it is fine.
I haven't tried the dropdown on mobile however.

Reply:
Agree, I'll take a look at that issue.